velox/connectors/hive/HiveConfig.h

```diff
@@ -147,6 +158,14 @@ class HiveConfig {
   static int32_t maxCoalescedDistanceBytes(const Config* config);
 
   static int32_t numCacheFileHandles(const Config* config);
+
+  static uint64_t const getKOrcWriterMaxStripeSize(
+      const Config* connectorQueryCtxConfig,
```

Comment:
I think it is better to merge these two configs at Prestissimo side so Velox only see one config.